apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -286,37 +288,26 @@ export default function SectionsSetUpContainer({
           )}
         </div>
       </div>
-
       <div
         className={classnames(
-          moduleStyles.buttonsContainer,
+          moduleStyles.splitButtonsContainer,
           moduleStyles.containerWithMarginTop
         )}
       >
-        {/*
-         TODO: for the first iteration of this feature we will only have
-        participants create one section at a time.  For edit section redirects,
-        adding another section is not needed.  This can be uncommented when the
-        functionality of adding another class section is ready.
-
-        {isNewSection && (
-        <Button
-          useDefaultLineHeight
-          icon="plus"
-          text={i18n.addAnotherClassSection()}
-          color={Button.ButtonColor.neutralDark}
-          onClick={e => {
-            e.preventDefault();
-            console.log('Add Another Class Section clicked');
-          }}
-        />
-        */}
-        {/*
-        TODO: currently this button just changes text if it is a "new" or "editied"
-        screen, depending on how we want the functionality of this button to work,
-        this might mean creating a different button for the "edit" page
-        */}
+        {isNewSection && ( // For edit section redirects, adding another section doesn't apply.
```

Comment:
tiny tiny nit: should this comment just say "when editing section, adding another section doesn't apply"?